lib/primer/dsl/view_helper.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Primer
+  # :nodoc:
+  module DSL
```

Comment:
Since this is closely related to `Primer::ViewHelper` what are your thoughts on defining this as a child module (`Primer::ViewHelper::DSL`) of `ViewHelper` to avoid having two modules named `ViewHelper`?